plugins/modules/ios_banner.py

```diff
@@ -43,6 +43,11 @@
     - incoming
     - slip-ppp
     type: str
+  multiline_delimiter:
```

Comment:
it would be great If we show the use case of `multiline_delimiter` in the example.